packages/calcite-components/src/components/chip-group/chip-group.tsx

```diff
@@ -49,7 +49,13 @@ export class ChipGroup implements InteractiveComponent {
   /** Specifies the size of the component. Child `calcite-chip`s inherit the component's value. */
   @Prop({ reflect: true }) scale: Scale = "m";
 
-  /** Specifies the selection mode of the component. */
+  /**
+   * Specifies the selection mode of the component:
+   * - `"multiple"` allows any number of selections,
+   * - `"single"` allows only one selection,
+   * - `"single-persist"` allow and require one selected item,
+   * - `"none"` does not allow any selections.
+   */
```

Comment:
Small nitpick: _Believe_ in this case both the `:` and `-` / `,` and `-` would be shown together. Maybe remove the `-` throughout so only one character is displayed in the doc?